pkg/driver/bacnet/merge/emergency.go

```diff
@@ -21,9 +21,18 @@ import (
 	"github.com/vanti-dev/sc-bos/pkg/task"
 )
 
+// AlarmConfig allows configuring a specific bacnet point to become an Emergency if the
+// value read from that point matches the ActivePolarity.
+type AlarmConfig struct {
+	config.ValueSource
+	ActivePolarity int    `json:"activePolarity"` // e.g. 1 if the alarm is active high, 0 if the alarm is active low
+	AlarmReason    string `json:"alarmReason"`    // the reason of the alarm
+}
```

Comment:
I like this, however may I suggest a few tweaks:
1. I think it would be better to model the OK case rather than the Emergency/Active case - especially with BACnet there's usually a single OK state and anything else is a bad state.
2. With the above in mind, I think the name of the type (and config field) should be something like `Value` or `Match`. It'd be nice to be able to support matching against Binary and Analog values: "read the val, if it's anything other than 1 then EMERGENCY" kind of thing.